index.html

```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
           <input type="submit" value="Rechercher" />
         </form>
       </section>
+      <section id="filters">
+        <h4>Filtres</h4>
+        <ul>
+          <li><i class="fas fa-money-bill-wave"></i>Economique</li>
+          <li><i class="fas fa-child"></i>Familial</li>
+          <li><i class="fas fa-heart"></i>Romantique</li>
+          <li><i class="fas fa-dog"></i>Animaux autorisés</li>
```

Comment:
Add link on word

Reply:
Is it only on the word or the whole line (word + icon) ?

Reply:
Juste to word

Reply:
done